src/scss/layout/_layout.scss

```diff
@@ -57,17 +43,59 @@ article {
 			display: table;
 		}
 	}
-}
 
-/*
- * Default WP Alignment Classes
+	/*
+ * Default WP image Alignment Classes
```

Comment:
Please fix the comment: styles for WP selectors in this section are no longer default ones from WordPress but customized for the WeCount website.